presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -92,6 +100,19 @@ fun AgendaScreen(
             agendaScreenEvent(AgendaScreenEvent.OnDateChanged(localDate))
         }
     )
+
+    AgendaDropDownMenu(
+        modifier = Modifier.background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
+        shouldOpen = agendaScreenState.agendaDropdownItemId.isNotEmpty(),
+        onCloseDropdown = {
+            /* Empty the list of item ids so the isExpanded dropdown will close as the items are empty */
+            agendaScreenEvent(AgendaScreenEvent.OnShowDropdown(listOf()))
+        },
+        listOfMenuItemId = listOf(R.string.open, R.string.edit, R.string.delete),
+        onSelectedOption = { item ->
+            Log.d("AGENDA", "ITEM [ $item ]")
+        }
+    )
 }
```

Comment:
One way is to do that. Have a box that fillMaxSize and set the content alignment to where you want

Another way, is to add a Dropdown to each item on the agenda. This would make it so the dropdown will always show at the right height on the screen

Reply:
@MKiperszmid I tried that by having a box to wrap the content in the scaffold. However, this didn't seem to work and the dropdown menu still displays on bottom start. Any ideas?

Reply:
Something like this will show the dropdown up or below of `FloatingActionButton`:
```
Box {
    DropdownMenu(...)
    FloatingActionButton(...)
}
```

Reply:
@philipplackner Thanks, its working now and it positions correctly.